paddle/fluid/inference/tests/api/analyzer_bert_tester.cc

```diff
@@ -251,6 +251,7 @@ TEST(Analyzer_bert, transfer_scope_cache) {
     threads.emplace_back([&, i]() {
       std::getline(fin, line);
       ParseLine(line, &input);
+      if (is_static) platform::set_cur_mkldnn_session_id(1);
```

Comment:
Please add a TODO here, we will hide directly use of `platform::set_cur_mkldnn_session_id(1)` later.

Reply:
done